client/src/components/Requests/CurrentRequests/CurrentRequests.tsx

```diff
@@ -0,0 +1,63 @@
+import { Paper, Typography, Grid } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import useStyles from './useStyles';
+import RequestListing from '../RequestListing/RequestListing';
+import RequestData from '../../../interface/Request';
+
+interface Props {
+  requests: RequestData | null | undefined;
+}
+
+const CurrentRequests = ({ requests }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  if (requests) {
+    return (
+      <Grid className={classes.root} component={Paper} elevation={2}>
+        <Box>
+          <Box>
```

Comment:
Typography components are block level components by default, I don't think we need to wrap it in a Box component here, unless there's any other reason? :)

Reply:
Change made. I was using Typography in line like a Span.